app/src/main/java/com/vci/vectorcamapp/core/presentation/components/tooltip/TooltipDialog.kt

```diff
@@ -0,0 +1,57 @@
+package com.vci.vectorcamapp.core.presentation.components.tooltip
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.AlertDialog
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import com.vci.vectorcamapp.ui.extensions.colors
+import com.vci.vectorcamapp.ui.extensions.dimensions
+
+@Composable
+fun TooltipDialog(
+    isVisible: Boolean,
+    title: String? = null,
+    content: @Composable (() -> Unit)? = null,
```

Comment:
I always suggest having content be the last property for a composable, because I think how compose works is, the last property can be passed in using curly braces. Eg. This will enable you to do something like:

TooltipDialog(...) {
    // pass in value for last property here. If it is of type @Composable, it will allow you to render child composables inside it.
}